### PR TITLE
Windows Release Build Tutorial | Added 'Distribute your build' section

### PR DESCRIPTION
**Pull request by chanmosq**
Signed-off-by: chanmosq <75444793+chanmosq@users.noreply.github.com>
## Change summary

- Added 'Distribute your build' section, which covers how to zip and send your build folder to other Windows computers. 
- Minor fixes

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/packaging/windows-release-builds.md [hunk 1]
```diff
@@ -378,12 +384,33 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
...
+        Monolithic
+        |   DevTestProject.GameLauncher.exe
+        |   PhysXDevice64.dll
+        |   PhysXGpu_64.dll
+        |   
+        \---Cache
+            \---pc
+            engine.pak
+                
+2. Distribute the zip folder in your preferred method such as through a file storage service.
```
Comment (chanmosq) [inline content/docs/user-guide/packaging/windows-release-builds.md:405]:
```suggestion
2. Distribute the zip folder by using your preferred method such as through a file storage service. 
```

Comment (paucom) [reply content/docs/user-guide/packaging/windows-release-builds.md:405]:
```suggestion
1. Distribute the `.zip` file using your preferred method, such as a file storage service. 
```
* Changed "2." to "1.", [per our ordered list formatting guidelines](https://www.o3de.org/docs/contributing/to-docs/style-guide/format/#ordered-lists).
* Same changes as above for "zip folder".
* For a more natural flow, I recommend setting aside the example phrase ("such as...") using a comma.
* To sound slightly less formal, I think that we can drop the words "by" and "through".

File: content/docs/user-guide/packaging/windows-release-builds.md [hunk 2]
```diff
@@ -378,12 +384,33 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
...
+        |   PhysXDevice64.dll
+        |   PhysXGpu_64.dll
+        |   
+        \---Cache
+            \---pc
+            engine.pak
+                
+2. Distribute the zip folder in your preferred method such as through a file storage service. 
+
+3. To run the Game Launcher on the other Windows computers, it must have [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
```
Comment (paucom) [inline content/docs/user-guide/packaging/windows-release-builds.md:407]:
```suggestion
To run the Game Launcher on other Windows computers, they must have the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
```
* I recommend dropping the ordered list numbering here, as this line is more of a note than an actionable step.
  * (Optional) I think that its appearance as a standard paragraph is fine, but this sentence might make the most sense [formatted as a note](https://www.o3de.org/docs/contributing/to-docs/style-guide/shortcodes/#note).
* Dropped unnecessary "the" ("other Windows computers").
* Presuming that "it" is meant to refer to "Windows computers," I changed it to "they." **Confirm if that's accurate.**
* Added "the" in front of "[MSVC" ("the [MSVC] Redistributable"), per phrasing in the linked MS docs.

**Review by paucom — CHANGES_REQUESTED** [review]
Added editorial comments.

File: content/docs/user-guide/packaging/windows-release-builds.md [hunk 3]
```diff
@@ -378,12 +384,34 @@ Run `GameLauncher.exe` from your project game release layout, which is located i
...
+            \---pc
+            engine.pak
+                
+1. Distribute the `.zip` file using your preferred method, such as a file storage service. 
+
+Now others can download and unpack the `.zip` file and run the Game Launcher on their Windows computers.
+
+{{< note >}}
+To run the Game Launcher on the other Windows computers, they must have the [Microsoft Visual C++ (MSVC) Redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-160) installed.
+{{< /note >}}
```
Comment (chanmosq) [inline content/docs/user-guide/packaging/windows-release-builds.md:409]: @paucom  I formatted the text in a note as you suggested. I also added the sentence "Now others can download..." to conclude the steps in this section.

**Review by paucom — APPROVED** [review]

**Review by sptramer — APPROVED** [review]

**Pull request merged**